tests/FunctionPathTest.php

```diff
@@ -0,0 +1,30 @@
+<?php
+/**
+ * @copyright 2018 Hostnet B.V.
+ */
+declare(strict_types=1);
+
+class FunctionPathTest extends \PHPUnit\Framework\TestCase
+{
+    public function testRun()
+    {
+        $current_location = getcwd();
```

Comment:
This seems like flaky behavior, imo it's better to use the location of this file as base location and resolve it from there (can use `__DIR__`)